dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/src/io/openliberty/security/jakartasec/fat/configs/TestConfigMaps.java

```diff
@@ -119,7 +176,105 @@ public static Map<String, Object> getEmptyCallerGroupsClaim() throws Exception {
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getRedirectURIEmpty_Logout() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.LOGOUT_REDIRECT_URI, Constants.EMPTY_VALUE);
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getNotifyProviderExpressionTrue() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.NOTIFY_PROVIDER_EXPRESSION, String.valueOf(true));
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getNotifyProviderExpressionFalse() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.NOTIFY_PROVIDER_EXPRESSION, String.valueOf(false));
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getAccessTokenExpiryExpressionTrue() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.ACCESS_TOKEN_EXPIRY_EXPRESSION, String.valueOf(true));
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getAccessTokenExpiryExpressionFalse() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.ACCESS_TOKEN_EXPIRY_EXPRESSION, String.valueOf(false));
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getIdentityTokenExpiryExpressionTrue() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.IDENTITY_TOKEN_EXPIRY_EXPRESSION, String.valueOf(true));
+        return updatedMap;
+
+    }
+
+    public static Map<String, Object> getIdentityTokenExpiryExpressionFalse() throws Exception {
+
+        Map<String, Object> updatedMap = new HashMap<String, Object>();
+        updatedMap.put(Constants.IDENTITY_TOKEN_EXPIRY_EXPRESSION, String.valueOf(false));
+        return updatedMap;
+
+    }
+
+//    public static Map<String, Object> mergeMaps(Map<String, Object>... indivMaps) throws Exception {
+//
+//        Map<String, Object> updatedMap = new HashMap<String, Object>();
+//        for (Map<String, Object> aMap : indivMaps) {
+//            for (Entry<String, Object> x : aMap.entrySet()) {
+//                updatedMap.put(x.getKey(), x.getValue());
+//            }
+//        }
+//        return updatedMap;
+//
+//    }
+
+    public static Map<String, Object> mergeMaps(Map<String, Object> updatedMap, Map<String, Object> newMap) throws Exception {
+
+        if (updatedMap == null) {
+            updatedMap = new HashMap<String, Object>();
+        }
+//        for (Map<String, Object> aMap : newMap.entrySet()) {
+        for (Entry<String, Object> x : newMap.entrySet()) {
+            updatedMap.put(x.getKey(), x.getValue());
+        }
+//        }
```

Comment:
```suggestion
        updatedMap.putAll(newMap);
```
I'm not sure this necessarily needs to be its own method, unless you just want one that will instantiate a new Map for you when `updatedMap` is null.

Reply:
I wanted a method to merge a map into another without having to worry if the first map existed (not having to check for that in a ton of locations, ...)
I updated the method to use putAll - will deliver with refresh tests